packages/graphql-migrations/src/connector/read.ts

```diff
@@ -92,13 +93,13 @@ class Reader {
           const columnName = this.getColumnName(key)
           if (!columnName) { continue }
           const info = columnInfo[key];
-          const {args, type} = getTypeAlias(info.type, info.maxLength);
+          const { args, type } = getTypeAlias(info.type, info.maxLength);
           const foreign = foreignKeys.find((k: any) => k.column === key);
-          const isPrimaryKey  = primaries.some((primary:  {column: string, indexName: string }) => primary.column === columnName);
+          const isPrimaryKey = primaries.some((primary: { column: string, indexName: string }) => primary.column === columnName);
           const column: TableColumn = {
             name: columnName,
             isPrimaryKey,
-            autoIncrementable: type === "increments" || type === "bigIncrements", // TODO
+            autoIncrementable: isAutoIncrementableColumn(this.config.client.toString(), info),
```

Comment:
```suggestion
            autoIncrementable: isPrimaryKey && isAutoIncrementableColumn(this.config.client.toString(), info),
```

Reply:
What additional benefit will this bring? Non-primary key columns can also be auto-incrementing, so adding this check would ignore that.

Reply:
You are right, but in our case I thought the auto-increment-ability was only handled for primary keys (worth checking) hence my suggestion. My worry is "if we allow other columns (from dbs) to be auto-incrementable while they are not when parsed from schema" could open another set of bugs with the diff algorithm.

Reply:
Yes, you are right about this. Thanks :+1: